EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/product_description/templates_and_design.md

# Templates and design

eZ Commerce comes with a set of templates and a standard design.

1. Our custom frontend Framework is based on [Foundation 5](http://foundation.zurb.com/).
2. We use [Sass](http://sass-lang.com/) (SCSS) to help us speed up the work when it comes to CSS.
3. We use [Twig](http://twig.sensiolabs.org/) as our templating language.

> 4. We use mostly [jQuery](http://jquery.com/) and a bit of [Backbone.js](http://backbonejs.org/) when it comes to JavaScript. Sometime we use plain vanilla JavaScript as well.

Reviewer: ```suggestion
4. We use mostly [jQuery](http://jquery.com/) and a bit of [Backbone.js](http://backbonejs.org/) when it comes to JavaScript. We also use plain vanilla JavaScript.
```
